src/pretix/base/models/event.py

```diff
@@ -1454,10 +1467,10 @@ def _seats(self, ignore_voucher=None):
         return qs_annotated
 
     @classmethod
-    def annotated(cls, qs, channel='web'):
+    def annotated(cls, qs, channel='web', voucher=None):
         from .items import SubEventItem, SubEventItemVariation
 
-        qs = super().annotated(qs, channel)
+        qs = super().annotated(qs, channel, voucher=voucher)
```

Comment:
It works named, but to make it more consistent:
```suggestion
        qs = super().annotated(qs, channel, voucher)
```

Reply:
Yeah, but using more keyword arguments generally makes the code easier to maintain in the future